i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-docker-restapi.md

```diff
@@ -67,72 +102,343 @@ ExecStart=/usr/bin/dockerd -H fd:// --containerd=/run/containerd/containerd.sock
+| CRITICALCONTAINERSSTOPPED | Critical threshold                                                                                                                                                                                                       |                                                                |             |
+| CRITICALNODESTATUS        | Define the conditions to match for the status to be CRITICAL (default: '%{status} !~ /ready/ \|\| %{manager\_status} !~ /reachable\|-/'). You can use the following variables: %{display}, %{status}, %{manager\_status} | %{status} !~ /ready/ \|\| %{manager\_status} !~ /reachable\|-/ |             |
+| WARNINGNODESTATUS         | Define the conditions to match for the status to be WARNING (default: -) You can use the following variables: %{display}, %{status}, %{manager\_status}                                                                  |                                                                |             |
+| EXTRAOPTIONS              | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).                                                                                                                       | --verbose                                                      |             |
+
+</TabItem>
+</Tabs>
+
+3. [Déployez la configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). Le service apparaît dans la liste des services supervisés, et dans la page **Statut des ressources**. La commande envoyée par le connecteur est indiquée dans le panneau de détails du service : celle-ci montre les valeurs des macros.
+
+## Comment puis-je tester le plugin et que signifient les options des commandes ?
+
+Une fois le plugin installé, vous pouvez tester celui-ci directement en ligne
+de commande depuis votre collecteur Centreon en vous connectant avec
+l'utilisateur **centreon-engine** (`su - centreon-engine`). Vous pouvez tester
+que le connecteur arrive bien à superviser une ressource en utilisant une commande
+telle que celle-ci (remplacez les valeurs d'exemple par les vôtres) :
 
 ```bash
-/usr/lib/centreon/plugins//centreon_docker_restapi.pl \
---plugin=cloud::docker::restapi::plugin \
---mode=node-status \
---hostname='192.168.0.50' \
---port='2375'   \
---warning-node-status='' \
---critical-node-status='%{status} !~ /ready/ || %{manager_status} !~ /reachable|-/' \
---verbose
-
-OK: Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0 | 'containers_running'=7;;;0; 'containers_stopped'=2;;;0; 'containers_paused'=0;;;0;
-Node '192.168.0.50' Containers Running : 7, Containers Stopped : 2, Containers Paused : 0
+/usr/lib/centreon/plugins/centreon_docker_restapi.pl \
+	--plugin=cloud::docker::restapi::plugin \
```

Comment:
is the example ok?

Reply:
Let's put the old one (I can't test this locally)